src/routes/(app)/namespaces/[namespace]/batch-operations/[jobId]/+layout.ts

```diff
@@ -0,0 +1,13 @@
+import { describeBatchOperation } from '$lib/services/batch-service';
+import { batchOperation } from '$lib/stores/batch-operations';
+
+import type { LayoutLoad } from '../$types';
+
+export const load: LayoutLoad = async ({ params, fetch }): Promise<void> => {
+  const operation = await describeBatchOperation(
+    { namespace: params.namespace, jobId: params.jobId },
+    fetch,
+  );
+
+  batchOperation.set(operation);
```

Comment:
Why not just return the operation and use $page.data.operation?

Reply:
Also, we'll need to remember to add these +layout files to cloud. which is totally fine

Reply:
Yeah true, we could do that. This does make auto-refresh easier cause then I can just update this store value. I'll tinker with it a little.

Reply:
Moved to using page.data and it works fine 👍